docs/en/changelog.md

```diff
@@ -5,7 +5,8 @@
 #### Compatibility
 
 - We migrate all the mmdet3d ops to mmcv and do not need to compile them when installing mmdet3d.
-- To fix the unprecise timestamp and optimize its saving method, we reformat the point cloud data during Waymo data conversion. The data conversion time is also optimized significantly by supporting parallel processing. Please re-generate KITTI format Waymo data if necessary. See more details in the [compatibility documentation](https://github.com/open-mmlab/mmdetection3d/blob/master/docs/en/compatibility.md).
+- To fix the imprecise timestamp and optimize its saving method, we reformat the point cloud data during Waymo data conversion. The data conversion time is also optimized significantly by supporting parallel processing. Please re-generate KITTI format Waymo data if necessary. See more details in the [compatibility documentation](https://github.com/open-mmlab/mmdetection3d/blob/master/docs/en/compatibility.md).
+- We update part of model checkpoints after the coordinate systems have been refactored. See more details in the #1369.
```

Comment:
part of model checkpoints after the coordinate systems have been refactored. -> some of model checkpoints after the refactor of coordinate systems. Please stay tuned for the release of the remaining model checkpoints.

Reply:
#1369 may need a hyperlink for reference here.